convert/gcode/toGcode.js

```diff
@@ -1,15 +1,18 @@
-import { op, transformCoordinate, visit } from '@jsxcad/geometry';
+import { linearize, transformCoordinate } from '@jsxcad/geometry';
 
 // FIX: This is actually GRBL.
-export const toGcode = async (geometry, { jumpHeight = 1 } = {}) => {
+export const toGcode = async (
+  geometry,
+  { speed = 0, feedrate = 0, jumpHeight = 1 } = {}
+) => {
   const codes = [];
 
   // CHECK: Perhaps this should be a more direct modeling of the GRBL state?
   const state = {
     // Where is the tool
-    x: undefined,
-    y: undefined,
-    z: undefined,
+    x: 0,
+    y: 0,
+    z: 0,
     // How 'fast' the tool is running (rpm or power).
     s: 0,
     f: 0,
```

Comment:
These defaulting to zero is probably the safest bet, but I have seen situations where if F defaults to zero when you try to run a program that doesn't set F the machine locks up and never moves...which is probably the safest result, but it's confusing since it seems like the machine is broken.

Reply:
Hmm, interesting.

I've cleaned this up a bit to only set to a changed F on demand of a cut operation.

Let's see if that will work better.